src/home/devrel-tracker.ts

```diff
@@ -1,22 +1,30 @@
+import { db } from "../../supabase";
+
 export function initiateDevRelTracking() {
-  const oldDevRelCode = localStorage.getItem("devRel");
+  const oldDevRelCode = localStorage.getItem("ref");
   if (!oldDevRelCode) {
     const urlParams = new URLSearchParams(window.location.search);
-    const devRelCode = urlParams.get("devRel");
+    const devRelCode = urlParams.get("ref");
     if (devRelCode) {
-      localStorage.setItem("devRel", devRelCode);
+      localStorage.setItem("ref", devRelCode);
```

Comment:
> One thing that I Just noticed is that in the URL https://work.ubq.fi/?devRel=XYZ the query parameter devRel is case-sensitive. The R in devRel must be capitalized. It would have been nice to have it case-insensitive but this is what is working now. 

A comment from the original implementation PR @gentlementlegen is this something we should consider? I'm unsure what the referral url schema looks like beyond `ref=`

Reply:
I think `ref` is sufficient for our needs here.

Reply:
sorry @0x4007 I meant what comes after `ref=`, the code or string or whatever the schema is, is case sensitivity a concern there? I presume no with the above but I haven't seen any campaign link generating first-hand so I wanted to check, ty.